[2023-07-19 04:58:44,092] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla scheduled__2023-07-18T00:00:00+00:00 [queued]>
[2023-07-19 04:58:44,189] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla scheduled__2023-07-18T00:00:00+00:00 [queued]>
[2023-07-19 04:58:44,191] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-19 04:58:44,193] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-19 04:58:44,195] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-19 04:58:44,265] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): crear_tabla> on 2023-07-18 00:00:00+00:00
[2023-07-19 04:58:44,279] {standard_task_runner.py:52} INFO - Started process 13288 to run task
[2023-07-19 04:58:44,315] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'crear_tabla', 'scheduled__2023-07-18T00:00:00+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_paises.py', '--cfg-path', '/tmp/tmphb8d5y2j', '--error-file', '/tmp/tmpwu1c3xr1']
[2023-07-19 04:58:44,318] {standard_task_runner.py:80} INFO - Job 69: Subtask crear_tabla
[2023-07-19 04:58:45,390] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.crear_tabla scheduled__2023-07-18T00:00:00+00:00 [running]> on host bd4a51b00e09
[2023-07-19 04:58:45,996] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Belenpintor
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=crear_tabla
AIRFLOW_CTX_EXECUTION_DATE=2023-07-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-18T00:00:00+00:00
[2023-07-19 04:58:46,008] {logging_mixin.py:115} INFO - creando conexion
[2023-07-19 04:58:46,288] {script_etl.py:20} INFO - DB_USER probando usuario 2: bapintor_coderhouse
[2023-07-19 04:58:46,295] {logging_mixin.py:115} INFO - usuario probando 2 bapintor_coderhouse
[2023-07-19 04:58:48,117] {logging_mixin.py:115} INFO - Conexi√≥n exitosa a la base de datos
[2023-07-19 04:58:48,628] {python.py:173} INFO - Done. Returned value was: None
[2023-07-19 04:58:49,708] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=crear_tabla, execution_date=20230718T000000, start_date=20230719T045844, end_date=20230719T045849
[2023-07-19 04:58:50,246] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-19 04:58:51,067] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
